lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -842,19 +848,27 @@ def process(
             the dataset have been reordered in comparison to the input
             percentiles.
         """
+        if isinstance(post_processed_forecast, iris.cube.Cube):
```

Comment:
Should this test for an iris.cube.CubeList ? It seems to be selecting the first entry in a cube otherwise. I assume the else case is meant to be for a single cube.